app/views/facility_orders/index.html.haml

```diff
@@ -34,6 +37,10 @@
               %td.centered= check_box_tag("order_detail_ids[]", od.id, false, {class: "toggle", id: nil })
               %td.centered= link_to od.order_id, facility_order_path(current_facility, od.order)
               %td.centered= link_to od.id, manage_order_detail_path(od), class: "manage-order-detail"
+              - if cross_core_order_view_enabled
+                %td
+                  - if od.order.cross_core_project.present? && od.order.cross_core_project.facility != current_facility
+                    = od.order.cross_core_project.facility
```

Comment:
Given clients said they don't want the Ordered By person I figured we may not need the icon and hover, so I started with this simpler version